Dockerfile

```diff
@@ -82,7 +86,12 @@ RUN apt-get update \
         zstd \
     && rm -rf /var/lib/apt/lists/*
 
-RUN ln -s /usr/bin/tini /sbin/tini
-COPY src/monobase /opt/r8/monobase
+COPY --from=build /src/dist /tmp/dist
+RUN ln -sv /usr/bin/tini /sbin/tini \
```

Comment:
Can the COPY+RUN be replaced by a `RUN --mount=type=bind,from=...` to eliminate the tmp layer?